packages/design-system/src/components/OcResource/OcResource.vue

```diff
@@ -229,6 +242,32 @@ export default defineComponent({
     max-height: $oc-size-icon-default * 1.5;
     width: $oc-size-icon-default * 1.5;
     max-width: $oc-size-icon-default * 1.5;
+
+    &-status-badge {
```

Comment:
this is because we need a different status badge for preview items (since its not an normal file icon)